ArduCopter/Attitude.cpp

```diff
@@ -307,3 +307,12 @@ void Copter::rotate_body_frame_to_NE(float &x, float &y)
     x = ne_x;
     y = ne_y;
 }
+
+// It will return the PILOT_SPEED_DN value if non zero, otherwise if zero it returns the PILOT_SPEED_UP value.
+uint16_t Copter::get_pilot_z_max_dn_velz()
+{
+    if(g2.pilot_velocity_z_max_dn == 0)
```

Comment:
Style rules: space after *if* and you need braces

Reply:
Updated in latest commit